docs/book/src/developer/providers/getting-started/controllers-and-reconciliation.md

```diff
@@ -1,5 +1,7 @@
 # Controllers and Reconciliation
 
+Right now, you can create objects with our API types, but you won't do anything about it. Let's fix that.
```

Comment:
" but you won't do anything about it." sounds a bit off

Reply:
fixed, PTAL